src/main/java/pl/agawesolowska/ticketbookingapp/model/TicketType.java

```diff
@@ -0,0 +1,57 @@
+package pl.agawesolowska.ticketbookingapp.model;
+
+import java.math.BigDecimal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Getter;
+
+/**
+ * @author Aga Wesołowska
+ *
+ */
+@Getter
+public enum TicketType {
+
+	@JsonProperty("adult")
+	ADULT(BigDecimal.valueOf(25)),
+	
+	@JsonProperty("student")
+	STUDENT(BigDecimal.valueOf(18)),
+	
+	@JsonProperty("child")
+	CHILD(BigDecimal.valueOf(12.50));
+
+	private BigDecimal price;
+
+	private TicketType(BigDecimal price) {
+		this.price = price;
+	}
+
+	public void setWeekendPrice(TicketType type) {
+		switch (type) {
+		case ADULT:
+			this.price = BigDecimal.valueOf(29);
+			break;
+		case STUDENT:
+			this.price = BigDecimal.valueOf(22);
+			break;
+		case CHILD:
+			this.price = BigDecimal.valueOf(16.50);
+			break;
+		}
+	}
+
+	public String toString(TicketType type) {
```

Comment:
czy to na pewno powinien być toString? ta metoda bardziej wyłuskuje kwotę ze środka?

Reply:
Pewnie lepsze byłoby: ticketValueToString